File: swift/28.swift
var sum = 1
var counter = 1
let x = 1001

for i in 1...(x - 1) / 2 {
  for _ in 1...4 {
    counter += 2 * i
    sum += counter
  }
}

print(sum)
// 669_171_001

File: solutions/28.c
#include <stdio.h>

int main() {
  int sum = 1;
  int counter = 1;
  int x = 1000;

  for (int i = 1; i <= x / 2; i++) {
    for (int j = 1; j <= 4; j++) {
      counter += 2 * i;
      sum += counter;
    }
  }

  printf("%d\n", sum);
  return 0;
}
// 669171001
